### PR TITLE
ENH Ensure users are sent emails when passwords are changed by default

### PR DESCRIPTION
**Pull request by GuySartorelli**
Also a fix to remove the backslash that was being output in emails, and fixing some markdown linting issues in the changelog since I was already editing that file.

Note that each of the three commits has a different prefix for changelog purposes - no need to squash.

## Parent issue
- #10306

### CODE REVIEW

**Comment by michalkleiner** [comment]
Looks good.

What was the rule around changing lang files through Transifex only? Does it also apply to default English files?

**Comment by GuySartorelli** [comment]
I'm not sure, tbh... I think in this specific case it should be done here, since the translations were already pulled from transifex for this release as part of the beta. But I'm not sure what the general rule is for the default en files.

File: docs/en/04_Changelogs/4.11.0.md [hunk 1]
```diff
@@ -89,6 +92,19 @@ SilverStripe\CMS\Model\SiteTree:
 New installations using `silverstripe/silverstripe-installer` and `silverstripe/recipe-kitchen-sink` from 4.11 onwards will have the above plugins added to the `allowed-plugins` configuration by default.
 - From July 2022 composer will no longer prompt to allow plugins when running `composer install`. This won't affect new installs using silverstripe/installer or silverstripe-recipe-kitchen-sink, but will affect other new projects, and existing projects where `allowed-plugins` hasn't yet been defined. In those cases developers will need to declare the allowed plugins manually in the project's `composer.json` file.
 
+### Users will recieve an email if their password is changed {#change-password-email}
+
+The `SilverStripe\Security\Member.notify_password_change` configuration has been set to `true` by default - it used to be `false`. This means when a user changes their password on a project in "live" mode, they will recieve an email alerting them that their password was changed. The email includes a link to change their password again, so that users can recover their account in the event that someone else changed their password without their knowledge.
```
Comment (emteknetnz) [inline docs/en/04_Changelogs/4.11.0.md:97]: Mention that sites with the `cwp/cwp-core` module installed already will already have the `notify_password_change` set the `true`

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Just need to mention that there will be no behavioral change on sites with cwp/cwp-core installed

Also, would prefer that we only tried to fix one thing at a time.  Even though it's more overhead, it's better to create a multiple PRs that each do a single thing.

**Comment by GuySartorelli** [comment]
> would prefer that we only tried to fix one thing at a time. Even though it's more overhead, it's better to create a multiple PRs that each do a single thing.

@emteknetnz Fair enough - so you want three separate PRs for this? Or do you just want the changelog linting separate (since the other two are related to each other, but that's kinda separate)?

**Comment by emteknetnz** [comment]
The changelog modifications unrelated to the bug that's being fixed should be in its own PR

Seems OK to keep the translation fix here since that's related to change password email that's being fixed

**Comment by GuySartorelli** [comment]
Linting PR: #10314

File: docs/en/04_Changelogs/4.11.0.md (hunk 1 above)
Comment (GuySartorelli) [reply docs/en/04_Changelogs/4.11.0.md:95]: Done

**Comment by kinglozzer** [comment]
While you’re in there, can you throw in a `$this->extend('updatePasswordChangedEmail', $email);` hook at the point the email is sent? These emails are a pain in the backside to modify otherwise!

**Comment by kinglozzer** [comment]
We typically run our email templates through a pair of templates: a wrapper that renders the `<head>` and some conditional tags/compatibility stuff for old email clients, then a `$Layout` variable that renders the email content. Essentially it allows us to do this and saves duplicating a tonne of code across multiple email templates:

```php
$html = $data->renderWith(['App\Email\ForgotPasswordEmail', 'App\Email\Email']);
$email->setBody($html);
```

**Comment by GuySartorelli** [comment]
That's fair - but should probably be done in a separate PR, since you'd probably want to add an extension point for the forgot email password as well - and maybe others.

**Comment by michalkleiner** [comment]
Separate PR would be better.

**Review by michalkleiner — APPROVED** [review]

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**